src/api/app/components/bs_request_state_badge_component.rb

```diff
@@ -7,7 +7,40 @@ def initialize(bs_request:, css_class: nil)
   end
 
   def call
-    tag.span(@bs_request.state,
-             class: ['badge', "bg-#{helpers.request_badge_color(@bs_request.state)}", @css_class])
+    content_tag(
+      :span,
+      tag.i(class: "fas fa-#{decode_state_icon(@bs_request.state)} me-1").concat(@bs_request.state),
+      class: ['badge', "bg-#{decode_state_color(@bs_request.state)}", @css_class]
+    )
+  end
+
+  def decode_state_color(state)
+    case state
+    when :review, :new
+      'secondary'
+    when :declined, :revoke
```

Comment:
I like the option to distinguish the `revoked` one actually, because it is not a real `danger` situation.
Maybe the dark one is too dark, I tried with something lighter and i like it so far. It looks like the following, what do you think?

![Screenshot from 2023-04-11 12-17-39](https://user-images.githubusercontent.com/7080830/231131367-73633a31-0d9c-48c5-a658-c5adc0789f1b.png)
![Screenshot from 2023-04-11 12-17-20](https://user-images.githubusercontent.com/7080830/231131373-29fdea0d-31e6-4c3b-b67a-667a4be6f9d1.png)

Reply:
I like the gray-ish one :+1: